WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingPaymentMethod/WooShippingPaymentMethodLine.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  WooShippingPaymentMethodLine.swift
+//  WooCommerce
+//
+//  Created by Rafael Kayumov on 11.05.2025.
+//  Copyright © 2025 Automattic. All rights reserved.
+//
+
```

Comment:
Super nit: We tend to remove the header when creating a new file. This pattern predates me, and I'm not sure why we do this, but it's best to follow the pattern for consistency with other files.